preview-env/comment/action.yml

```
@@ -0,0 +1,51 @@
+---
+name: Manage Deployment Result Summary
+description: |
+  Manages a deployment result summary as comment in the current PR based on deployment artifacts.
+  If deployment artifacts have been uploaded, the action upserts the summary.
+  If there are no deployment artifacts, the summary will get deleted.
+  The deployment artifact name pattern: `deployment-status-<runID>-<runAttempt>-*`
+runs:
+  using: composite
+  steps:
+    - name: Download Deployment Status Artifacts
+      uses: actions/download-artifact@v4
+      with:
+        path: deployment_status
+        pattern: deployment-status-${{ github.run_id }}-${{ github.run_attempt }}-*
+    - name: Check for deployment artifacts
```

To be consistent with the step name above
```suggestion
    - name: Check for Deployment Status Artifacts
```